taxi/migrations/0001_initial.py

```diff
@@ -29,7 +29,7 @@ class Migration(migrations.Migration):
                 ('last_name', models.CharField(blank=True, max_length=150, verbose_name='last name')),
                 ('email', models.EmailField(blank=True, max_length=254, verbose_name='email address')),
                 ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
-                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
```

Comment:
do you need this change?
you should use auto replace carefully and check what it changes before applying changes because migrations may fail ;)

Reply:
Yeah, it was casually